deploy/examples/create-external-cluster-resources.py

```diff
@@ -1152,6 +1152,12 @@ def create_rgw_admin_ops_user(self):
                     "info",
                     "--uid",
                     self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
+                    "--rgw-realm",
```

Comment:
@thotz can you confirm this,
Do we need to pass the flag if a user is already created?